components/style/values/mod.rs

```diff
@@ -104,6 +104,19 @@ pub enum Either<A, B> {
     Second(B),
 }
 
+impl<A, B> Either<A, B> {
+    pub fn is_first(&self) -> bool {
```

Comment:
We had these methods earlier, but ended up removing them, because they give vague/no information about the underlying type (which is the important downside of `Either`). Instead, we could just use `if let` patterns wherever possible to remind ourselves about `Either` type. Anyway, please remove this `impl`